src/results/rangeTable.tsx

```diff
@@ -15,7 +15,7 @@ import { ScatteringOptions, useResultStore } from "./resultsStore";
 import { ReciprocalWavelengthUnits, WavelengthUnits } from "../utils/units";
 import {
   convertBetweenQAndD,
```

Comment:
maybe: could this 'between' be also renamed to 'to', like on line 18? it's clearer